src/app/common/app.helpers.ts

```diff
@@ -1,7 +1,11 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
```

Comment:
Never write global pragmas. We discussed this many many times.

Reply:
Try fixing the problem instead, by improving typing or a form of expressions.
In the worst case, with a local pragma, if the issue is so unsolvable.

Reply:
Fixed